sdk/digitaltwins/azure-digitaltwins-core/azure/digitaltwins/core/aio/__init__.py

```diff
@@ -7,5 +7,15 @@
 # --------------------------------------------------------------------------
 
 from ._digitaltwins_client_async import DigitalTwinsClient
+from ._models import DigitalTwinsEventRoute
+from .._generated.models import DigitalTwinsModelData
+from .._generated.models import QueryResult
+from .._generated.models import IncomingRelationship
```

Comment:
These models shouldn't be added to the `aio` namespace.

Reply:
Removed.